grazer/src/main/java/edu/asu/diging/grazer/core/domain/IFileData.java

```diff
@@ -0,0 +1,21 @@
+package edu.asu.diging.grazer.core.domain;
+
+import java.util.List;
+
+/**
+ * 
+ * This class stores the file name and file data which is in byte format.
```

Comment:
for what files?

Reply:
also, it's called FileData but has a list of filenames. does it represent one or multiple files?

Reply:
and my previous comment:
"Also, instead of using a list of filesname and a list of corresponding byte arrays, it would be much less error prone to have a list of file objects (whatever you want to call the class) that map filename to byte array."